scripts/docker/buildpack-deps/Dockerfile.ubuntu1804

```diff
@@ -84,20 +84,20 @@ RUN set -ex; \
 	EVMONE_VERSION="$EVMONE_MAJOR.$EVMONE_MINOR.$EVMONE_MICRO"; \
 	TGZFILE="evmone-$EVMONE_VERSION-linux-x86_64.tar.gz"; \
 	wget https://github.com/ethereum/evmone/releases/download/v$EVMONE_VERSION/$TGZFILE; \
-	sha256sum $TGZFILE; \
+	sha256sum $TGZFILE | grep ${EVMONE_HASH}; \
 	tar xzpf $TGZFILE -C /usr; \
 	rm -f $TGZFILE;
 
 # HERA
-ARG HERA_HASH="622663cb3bc1fa07213c6e1fe8070c5c259096e75039a46d014b2a3448b5cf44"
+ARG HERA_HASH="1a0b3cf626910c969f0ac86b7a731969a2e5b8254bc4e626b8f3a60471481f03"
 ARG HERA_MAJOR="0"
 ARG HERA_MINOR="3"
-ARG HERA_MICRO="0"
+ARG HERA_MICRO="2"
 RUN set -ex; \
 	HERA_VERSION="$HERA_MAJOR.$HERA_MINOR.$HERA_MICRO"; \
 	TGZFILE="hera-$HERA_VERSION-linux-x86_64.tar.gz"; \
 	wget https://github.com/ewasm/hera/releases/download/v$HERA_VERSION/$TGZFILE; \
-	sha256sum $TGZFILE; \
+	sha256sum $TGZFILE | grep ${HERA_HASH}; \
```

Comment:
Hmm, what was the point of the original line? :)

Reply:
Don't know ;)

Reply:
I didn't even know that grep returns 0 or 1 exit codes whether it found something or not.